packages/graphql/mixin.core.js

```diff
@@ -14,6 +15,25 @@ function exists(path) {
   }
 }
 
+function contextRelativeExternals(externals) {
```

Comment:
I'd personally go for `... externals` here to not have to wrap those module names into an unnecessary array.